src/Raven.Client/Documents/Changes/ChangeNotification.cs

```diff
@@ -38,6 +38,11 @@ internal static TopologyChange FromJson(BlittableJsonReaderObject value)
         }
     }
 
+    public class AggressiveCacheUpdate : DatabaseChange
```

Comment:
can this be internal?

Reply:
AggressiveCacheChange to be consistent? not sure

Reply:
Done